lib/rules/require-computed-property-dependencies.js

```diff
@@ -412,9 +412,8 @@ module.exports = {
           if (undeclaredKeys.length > 0) {
             context.report({
               node,
-              message: `Use of undeclared dependencies in computed property: ${undeclaredKeys.join(
-                ', '
-              )}`,
+              message: `Use of undeclared dependencies in computed property: {{undeclaredKeys}}`,
+              data: { undeclaredKeys: undeclaredKeys.join(', ') },
```

Comment:
what is the advantage of this? 🤔

Reply:
This rule prefers placeholders!

https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/prefer-placeholders.md